hardware_interface/include/fake_components/generic_system.hpp

```diff
@@ -90,6 +96,9 @@ class HARDWARE_INTERFACE_PUBLIC GenericSystem : public hardware_interface::Syste
     std::vector<std::vector<double>> & commands, std::vector<std::vector<double>> & states,
     const std::vector<std::string> & interfaces);
 
+  void populate_gpio_interfaces();
+
+  bool fake_gpio_command_interfaces_;
```

Comment:
Maybe better:

```suggestion
  bool use_fake_gpio_command_interfaces_;
```

If you agree we should later rename the next line too. (but in a separate PR)

Reply:
Yes, it seems better. Maybe we will do it in the next enhancement PR altogether.